setup.py

```diff
@@ -0,0 +1,46 @@
+from setuptools import setup, find_packages
+import codecs
+import os
+
+here = os.path.abspath(os.path.dirname(__file__))
+
+with codecs.open(os.path.join(here, "README.md"), encoding="utf-8") as fh:
+    long_description = "\n" + fh.read()
+
+VERSION = '{{VERSION_PLACEHOLDER}}'

```

Comment:
Here we should add something I guess.  0.1?

Reply:
@ghiggi , This is to be dynamic. I'm going to show you that this afternoon, but the version on pypi will be linked to the tag version in github. Therefore this file should be unchnaged.